docs/fluvio/cli/fluvio/hub/download.mdx

```diff
@@ -42,8 +42,6 @@ downloading infinyon/json-sql@0.1.0 to infinyon-json-sql-0.1.0.ipkg
 3. Choose the SmartModule you want to download, pick a version, and click Download
     The SmartModule is downloaded to your cluster and the state of the button changes.
 
-[`the profile`]: /docs/fluvio/cli/fluvio/profile
+[`the profile`]: fluvio/cli/fluvio/profile.mdx
```

Comment:
We didn't port profile to the `cli`. Let's port it so this link resolves.

Reply:
this resolves